contracts/OctobayGovNFT.sol

```diff
@@ -139,7 +139,9 @@ contract OctobayGovNFT is OctobayStorage, ERC721Pausable {
         _revokePermission(_tokenId, Permission.TRANSFER);
         _revokePermission(_tokenId, Permission.SET_ISSUE_GOVTOKEN);
         _revokePermission(_tokenId, Permission.CREATE_PROPOSAL);
+        _unpause();
         _burn(_tokenId);
+        _pause();
         emit BurnTokenEvent(_tokenId);
     }
```

Comment:
I think we need some more checks here. Burning an NFT can only be done by the owner of that NFT or the owner of a permission NFT with the ADMIN permission for the governance token that the NFT to burn belongs to.

Reply:
Yeah sorry, this was lazy of me. I had in my mind that the `_burn()` itself would do those checks but it turns out the `_burn()` itself doesn't check this either though.

I feel like allowing admins to burn other users NFTs is getting a bit dicey though. It would mean one admin could burn other admins tokens. I'm not sure that's a good idea. Thoughts?